client/src/assets/css/main.css

```diff
@@ -0,0 +1,48 @@
+.text-center{
+    text-align: center;
+}
+
+.text-right{
+    text-align: right;
```

Comment:
Yet these classes aren't general.
A convention like
```css
.hidden {
  display: none;
}
```
is okay because it conveys meaning rather than the direct css. What I'm saying is your text-right --> text-align: right is about as good as inline css.